packages/react-native/src/Authenticator/Defaults/ConfirmResetPassword/ConfirmResetPassword.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import { Text } from 'react-native';
 
 import { ConfirmResetPasswordComponent } from './types';
+import { DefaultHeader } from '../../common/DefaultHeader';
 
 const ConfirmResetPassword: ConfirmResetPasswordComponent = () => {
   return <Text>ConfirmResetPassword</Text>;
 };
 
 ConfirmResetPassword.Header = function Header() {
```

Comment:
We should be able to just assign `DefaultHeader` as the value here:
```
ConfirmResetPassword.Header = DefaultHeader;
```

Passing of `children` will be a parent component concern